### DISCUSSION TITLE
v1.3.0

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#72] [Announcements]
## New Devices

- HXi and RMi series power supplies (HID PSU)
- AXi series power supplies (Flex USB PSU)
  - Note: Communication with an AXi PSU requires driver installation. Instructions are provided in `Corsair-SiUsbXpress-Driver/README.txt`.

## Existing Devices

- Commander CORE / Commander PRO / Coolit: Resolves an issue controlling devices after switching configurations (#58).

## Other

- General: Presents an alert if the incorrect build of Fan Control is detected.
- Logging: All logs are now written to `CorsairLink.log` instead of Fan Control's `log.txt` to prevent file contention.
- Logging: Presents an alert if multiple (10) errors are detected. This can be disabled by setting the `FANCONTROL_CORSAIRLINK_ERROR_NOTIFICATIONS_DISABLED` environment variable to `1` and restarting Fan Control.
- Logging: Updates logging infrastructure to match the Razer plugin.
- Logging: General logging improvements. Debug logging can be enabled by setting the `FANCONTROL_CORSAIRLINK_DEBUG_LOGGING_ENABLED` environment variable to `1` and restarting Fan Control.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/'>v1.3.0</a>.</em>

**Comment by xziste** [comment]
how to disable the error massage

**Reply by EvanMulawski** [reply]
In Windows environment variable settings, add `FANCONTROL_CORSAIRLINK_ERROR_NOTIFICATIONS_DISABLED` with a value of `1` and restart Fan Control. However, if you are seeing this, there is likely a device communication problem that needs to be addressed. Can you post the `CorsairLink.log` file so I can take a look?
